webrender/src/device/gfx/image.rs

```diff
@@ -19,6 +19,10 @@ const DEPTH_RANGE: hal::image::SubresourceRange = hal::image::SubresourceRange {
     layers: 0 .. 1,
 };
 
+/// The Vulkan spec states: bufferOffset must be a multiple of 4 for VkBufferImageCopy
+/// https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#VkBufferImageCopy
+const BUFFER_COPY_ALIGNMENT_MASK: i32 = 4;
```

Comment:
Just call this `BUFFER_COPY_ALIGNMENT`, because the mask value is this minus one.